plenum/server/request_handlers/state_constants.py

```diff
@@ -0,0 +1,2 @@
+MARKER_TAA = "2"
```

Comment:
Should we rename the file to `config_state_constants`?

Reply:
I think, no. We can store constants for domain and config state in this file.